swagger_server/controllers/operand_controller.py

```diff
@@ -22,7 +34,11 @@ def delete_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    try:
+        operand = Operand.delete().where(Operand.id == operand_id)
+    except Exception as exc:
+        raise ValidationError(400, 'Expression not found!')
+    return flask.jsonify({'expression_id': operand_id})
```

Comment:
I think you can just return a jsonified True here if the delete succeeded and a 404 validation error if it wasn't able to delete stuff.